### ISSUE TITLE
swiftUI error

### ISSUE THREAD

**Issue by buzai** [enebin/Aespa#21]
i use in readme code
```
       aespaSession
            .autofocusing(mode: .continuousAutoFocus)
            .orientation(to: .portrait)
            .quality(to: .high)
```
but it tell me
VideoContentViewModel.swift:137:14 Value of type 'AespaSession' has no member 'autofocusing'

**Comment by buzai** [comment]
If I copy the file in the demo to my project, it will tell me that the class of VideoAsset and PhotoAsset is missing

**Comment by StarboySupernova** [comment]
Same

**Comment by buzai** [comment]
@StarboySupernova 
hi, Can I use your fork directly?

**Comment by StarboySupernova** [comment]
> @StarboySupernova hi, Can I use your fork directly?

Certainly. Don't forget to keep monitoring the upstream repo so we can sync the changes, if necessary

**Comment by enebin** [comment]
Hi guys. I'm very sorry for the late response. You might have given up and found another solution by now 😅 

This issue has been resolved in the latest version (currently `0.4.2`). The _focusing_ method was changed once, and it was my fault for not updating the README. 

Thank you for reporting the issue, and I am waiting to hear back if the problem has been resolved 🥲
